src/HealthGPS/kevin_hall_model.cpp

```diff
@@ -81,74 +94,114 @@ void KevinHallModel::update_risk_factors(RuntimeContext &context) {
         if (person.age == 0) {
             initialise_nutrient_intakes(person);
             initialise_energy_intake(person);
-            initialise_weight(person);
         } else {
             update_nutrient_intakes(person);
             update_energy_intake(person);
         }
     }
 
-    // TODO: Compute target body weight.
-    const float target_BW = 100.0;
-
-    // Trial run.
-    double mean_sim_body_weight = 0.0;
-    double mean_adjustment_coefficient = 0.0;
+    // Update: different model for children and adults.
     for (auto &person : context.population()) {
         // Ignore if inactive.
         if (!person.is_active()) {
             continue;
         }
```

Comment:
Given that iterating over only the "active" people seems to be such a common operation, would it be useful to have some kind of helper function?

Obvs shouldn't be done as part of this PR

Reply:
Agree. Less bug prone. I'll open an issue.

Reply:
#267.